README.md

```diff
@@ -1,158 +1,270 @@
-
->The default settings are satisfactory, but you may modify them as desired.
-
-**Step 7:** After the installation is complete, open Vision Client Launcher if you have not done so already, and click `Add Vision Client`.
-
->For more information on location/opening Vision Client Launcher, see Usage, Step 1.
-
-**Step 8:** Double-click your Ignition gateway from the list of `On Your Network` gateways.
-
->Since your gateway is installed on the same computer as Vision Client Launcher, the URL field of your gateway should display `http://localhost:8088`.
-
->Please note that `8088` is a user-configurable port number, and your installation may not use port `8088` if you modified the port settings.
-
-**Step 9:** Ensure the Vision project you imported in Step 3 is selected, and click `Add Application(s)`.
-
->Please note, the imported Vision project may not use the name you chose earlier, and can be found as `HMS-Ewon-Ignition-Demo`.
-
-**Step 10:** [Optional] Click the down arrow, located to the right of `Launch Fullscreen`, and click `Create Shortcut` to add a demo shortcut to your desktop for convenience.
-
-## Usage ##
-
-**Step 1:** Launch Vision Client Launcher by clicking its icon in your operating system's main menu.
-
-* *Windows:* Start Button
-* *macOS:* Launchpad
-* *Linux:* Varies (commonly upper left corner)
-
-**Step 2:** Locate your Vision Project and click `Launch Fullscreen`.
-
-**Step 3:** Enter the username and password you set during Ignition commissioning and click `Login`.
-
-**Step 4:** Fill out the requested information, and if desired, choose `Remember Me` to allow for easy log in for subsequent demos.
-
->In the `Ewon Name` field, enter the exact name of the Ewon device you've configured for this demo, and click `Save`.
-
->Please note that you can locate the name of your Ewon device by browsing to your Ewon device's web page, expanding `Setup`, expanding `System`, clicking `Main` and selecting `Identification`.
-
-**Step 5:** Click `Begin` to start the demo.
-
-## Troubleshooting ##
-If you experience a problem or bug, please check the Ewon Ignition Demo wiki for more information by visiting [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "https://github.com/hms-networks/FlexyIgnitionDemo/wiki").
-
-If you are unable to find information on your issue, or your issue persists, please contact HMS Networks by emailing [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or create an issue on the Ewon Ignition Demo GitHub project by visiting [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "https://github.com/hms-networks/FlexyIgnitionDemo/issues").
+<!DOCTYPE html>
+<html>
+<body>
+  <!-- HEADER/NAME -->
+  <h1 id="name"><span style="font-variant: small-caps;">Ewon Ignition Demo</span></h1>
```

Comment:
did you mean to drop the copyright symbol?

Reply:
Yes. I created a trademark information section and pulled the official lines from their respective trademark holders. The symbols didn't carry over between HTML/Word/PDF consistently, so I'm minimizing the number of symbols, and isolating the ones that still are in the document.